plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx

```diff
@@ -28,19 +28,24 @@ import type { Case } from '../../types/types';
 import { BannerContainer, Text, CaseLink, BannerActionLink } from './styles';
 import InfoIcon from './InfoIcon';
 import { showRemovedFromCaseBannerAction } from '../../states/case/caseBanners';
+import { contactFormsBase, namespace } from '../../states/storeNamespaces';
+import { RootState } from '../../states';
 
 type OwnProps = {
   taskId: string;
+  contactId: string;
 };
 
 type Props = OwnProps & ReturnType<typeof mapStateToProps> & ReturnType<typeof mapDispatchToProps>;
 
-const mapStateToProps = (state, { taskId }: OwnProps) => {
+const mapStateToProps = (state: RootState, { taskId, contactId }: OwnProps) => {
   const contact = selectContactByTaskSid(state, taskId);
   const connectedCase = selectCaseByTaskSid(state, taskId);
+  const caseId = state[namespace][contactFormsBase].existingContacts[contactId].savedContact?.caseId;
```

Comment:
```suggestion
  const caseId = contact?.savedContact?.caseId;
```

Would this not work and allow the fix to be even simpler?